class-job.php

```diff
@@ -51,6 +51,8 @@ public function save() {
 			$data['interval'] = $this->interval;
 		}
 
+		wp_cache_delete( "jobs:{$this->site}", 'cavalcade-jobs' );
```

Comment:
@rmccue: I'm cool either way as long as we don't do 40 redundant database queries. Should I just just a variable?